packages/babel-preset-stage-3/index.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   plugins: [
-    require("babel-plugin-transform-async-to-generator"),
-    require("babel-plugin-transform-exponentiation-operator")
```

Comment:
If i'm not mistaken (again), removing exponentiation operator from stage-3 would be a breaking change - if we actually want to remove it at some point the least we can do is a deprecation message, otherwise do a major bump (in which case we would need to change how we release a bit)